pilot/pkg/model/push_context.go

```diff
@@ -1416,6 +1433,22 @@ func (ps *PushContext) initServiceRegistry(env *Environment) error {
 	return nil
 }
 
+// pre computes secrets per namespace
+func (ps *PushContext) initSecrets(env *Environment) error {
+	secretConfigs, err := env.List(gvk.Secret, NamespaceAll)
```

Comment:
Won't this get all secrets from all components? Including k8s?

Reply:
I think, no. It get configs from `model.ConfigStore`. but, seems the secrets from k8s, is on another path, `credentials.Controller`, which is not a kind of `model.ConfigStore`.